lib/Dashboard/Controller/API/Incidents.pm

```diff
@@ -37,6 +37,9 @@ my $INCIDENT_SPEC = {
     isActive    => {type  => 'boolean'},
     embargoed   => {type  => 'boolean'},
     priority    => {anyOf => [{type => 'integer'}, {type => 'null'}]},
+    scminfo     => {type  => 'string'},
+    url         => {type  => 'string'},
+    type        => {type  => 'string'},
```

Comment:
Do we need this `type` field? I feel we can distinguish based on different attributes, for example `scminfo` presence. Did I miss something?

Reply:
I would make it explicit and have a type. Everything else just feels like a hack that makes things more complicated, like having to interpret `scminfo`.

Reply:
I see, my issue with this is that so far we do not have any enum-like structure to define the "keys" for the type and `smelt` is kind of ok, but it is hard to find out the other values. `git` is probably not a good choice and I would probably prefer `gitea` which describe the data source like `smelt` is. What do you think?

Reply:
And code like `incident.type == …` is much more intuitive to read than code like `incident.scminfo != ''` based on the assumption that scminfo is empty for certain types of incidents.

Reply:
Yes, you are right, it is probably a non-issue as long as this is only used for displaying (or not-displaying) stuff.